ingests/ggdc_maddison_project_database.py

```diff
@@ -0,0 +1,274 @@
+"""This script should be manually adapted and executed on the event of an update of the Maddison Project Database.
+
+"""
+
+import argparse
+from datetime import datetime
+
+from owid.walden import Dataset
+
+# Date of publication of the dataset.
+# Since we don't know this exact date, we use the first day of the month of the publication of the paper
+# https://www.rug.nl/ggdc/historicaldevelopment/maddison/publications/wp15.pdf
+PUBLICATION_DATE = "2020-10-01"
+PUBLICATION_YEAR = 2020
```

Comment:
I've checked walden and you can provide either year or date, shouldn't matter. Either of those are then used as `version` of the dataset.